config.toml

```diff
@@ -123,7 +123,7 @@ url_latest_version = "https://kubeflow.org/docs/"
 
 # A variable used in various docs to determine URLs for config files etc.
 # To find occurrences, search the repo for 'params "githubbranch"'.
-githubbranch = "v0.7-branch"
+githubbranch = "v1.0-branch"
```

Comment:
it looks like this is referring to kubeflow/manfiests

I think some of the links need to be updated.

```
./content/docs/gke/customizing-gke.md:      * one for CPU only machines [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deployment_manager_configs/cluster.jinja#L109).
./content/docs/gke/customizing-gke.md:      * one for GPU machines [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deployment_manager_configs/cluster.jinja#L149).
./content/docs/gke/customizing-gke.md:  * When making changes to the node pools you also need to bump the pool-version [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deployment_manager_configs/cluster-kubeflow.yaml#L37) before you update the deployment
```

Those links on customizing GKE should now be
https://github.com/kubeflow/manifests/tree/v1.0-branch/gcp/deployment_manager_configs

As opposed to kubeflow/kubeflow

Reply:
Done.